### DISCUSSION TITLE
The  .msh file I generated with Gmsh is different from the  .msh   file in Sofa, may I ask if I need to do additional operations?

### DISCUSSION THREAD

**Discussion by MingSongJiang** [InfinyTech3D/SofaUnity#116] [Help - Using SofaUnity] [answered]
Hello,
The  .msh file I generated with Gmsh is different from the  .msh   file in Sofa, may I ask if I need to do additional operations?
Thanks!
![sublime_text_Hx7gklOYRh](https://github.com/user-attachments/assets/deb57f44-3333-4960-811f-d59d8f3558fe)

**Comment by MingSongJiang** [comment]
Use Gms3

**Comment by epernod** [comment]
Hi,

according to the 2nd line of the files, one is version 2.2 and the other is 4.1, that explains the difference.

Best regards,

**Reply by MingSongJiang** [reply, marked as answer]
Yes, I used Gmsh3 to generate the same file format

**Reply by epernod** [reply]
yes that's fine.
